test/js/data/fill_bucket.test.js

```diff
@@ -16,7 +17,7 @@ var feature = vt.layers.water.feature(0);
 function createFeature(points) {
     return {
         loadGeometry: function() {
-            return [points];
+            return points[0] instanceof Point ? [points] : points;
```

Comment:
I would rather edit the existing test cases to include an outer array than introduce this logic into our test cases.

(Logic in test fixture generation leads to bugs in tests. Bugs in tests leads to hair-pulling.)